todo-app.css

```diff
@@ -0,0 +1,60 @@
+*{
+    box-sizing: border-box;
+}
+
+:root {
+  --background: rgb(220, 248, 220);
+  --border: #1f5319 1.2px solid;
+}
+
+body {
+  background: var(--background);
+  margin: 0;
+  padding: 1ch;
```

Comment:
`ch` represent width of a `0` character. Why use that unit for padding? Whats your intent here?